libs/ui/src/diagnostics/configuration_section.tsx

```diff
@@ -128,7 +128,7 @@ export function ConfigurationSection({
       <section>
         <H2>Configuration</H2>
         <P>
-          <InfoIcon /> No election currently loaded on device.
+          <InfoIcon /> No election currently loaded on device
```

Comment:
<img width="366" alt="Screen Shot 2024-10-09 at 9 45 45 AM" src="https://github.com/user-attachments/assets/963f5ba6-415d-4653-8b28-998cb15ef53c">
<img width="373" alt="Screen Shot 2024-10-09 at 9 45 51 AM" src="https://github.com/user-attachments/assets/2c7f1068-6430-486c-a442-5067e8561203">

Reply:
don't need the word "currently"

Reply:
3c117e9c3